README.md

```diff
@@ -8,6 +8,16 @@
 
 The native RADOS library and development headers are expected to be installed.
 
+On debian:
+```sh
+sudo apt install -y libcephfs-dev librbd-dev librados-dev
+```
+
+On rhel:
+```sh
+sudo dnf install -y libcephfs-devel librbd-devel librados-devel
```

Comment:
i would not add `dnf` as pkg manager but only the name of pkgs ( there is `yum` and `zypper`)

Reply:
Along with the last comment, maybe something along the lines of:

On `yum, dnf, zypper` systems: `libcephfs-devel librbd-devel librados-devel`